src/lib/decorators.d.ts

```diff
@@ -17,8 +17,14 @@ type DecoratorContext =
     | ClassMemberDecoratorContext
     ;
 
+type DecoratorMetadataObject = Record<PropertyKey, unknown> & object;

```

Comment:
This will also need to be added to the global symbols that the fourslash tests expect.